test/electron/index.js

```diff
@@ -29,6 +29,14 @@ const optimist = require('optimist')
 
 const argv = optimist.argv;
 
+// {{SQL CARBON EDIT}}
+// Set test run options. These are NOT used if grep is specified manually - that implies the user has a specific desire to
+// filter the tests beyond the defaults set for ADS_TEST_GREP in the calling scripts.
+if (!argv.grep) {
```

Comment:
I generally use -g instead of --grep.

Reply:
Nvm i see its decribed above as both

Reply:
Yeah I tested this with -g so this should be working as intended with both 😄